src/components/MyKiva/LendingStats.vue

```diff
@@ -424,6 +427,11 @@ export default {
 					'close-goals'
 				);
 			}
+			if (this.isGoalSet && !this.recordedGoalSet) {
+				// eslint-disable-next-line max-len
+				this.$kvTrackEvent('portfolio', 'show', 'goal-set', this.newGoalPrefs.category, this.newGoalPrefs.target);
```

Comment:
We probably want optional chaining on `this.newGoalPrefs` just in case here for the two uses